cloudant-sync-datastore-android/src/test/java/com/cloudant/sync/replication/ReplicationServiceTest.java

```diff
@@ -682,4 +695,22 @@ public void testSetReplicatorsMultipleInvocations() {
         }
     }
 
+    private void checkClockTime(long expectedTime, long actualTime) {
+        assertTrue("Last alarm elapsed time not within " + ALARM_TOLERANCE_MS + "ms of expected time",
+                Math.abs(expectedTime - actualTime) < ALARM_TOLERANCE_MS);
+    }
+
+    private void checkElapsedTime(long expectedTime, long actualTime) {
+        assertTrue("Last alarm clock time not within " + ALARM_TOLERANCE_MS + "ms of expected time",
```

Comment:
More idiomatic would be to use a Hamcrest matcher like [closeTo](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matchers.html#closeTo(double,%20double))